src/main/java/gregtech/loaders/oreprocessing/ProcessingSaplings.java

```diff
@@ -15,6 +16,7 @@ public ProcessingSaplings() {
 
     public void registerOre(OrePrefixes aPrefix, Materials aMaterial, String aOreDictName, String aModName, ItemStack aStack) {
         GT_ModHandler.addPulverisationRecipe(GT_Utility.copyAmount(1L, new Object[]{aStack}), GT_OreDictUnificator.get(OrePrefixes.dustSmall, Materials.Wood, 2L), null, 0, false);
+        GT_ModHandler.addCompressionRecipe(GT_Utility.copyAmount(8L, new Object[]{aStack}), ItemList.IC2_PlantballCompressed.get(1L, new Object[0]));
```

Comment:
ItemList.IC2_PlantballCompressed? Isn't that "Compressed Plants" which isn't used anymore in IC2Exp? (see https://wiki.industrial-craft.net/index.php?title=Compressed_Plants )